extensions/interactions/EndExploration/EndExploration.js

```diff
@@ -73,9 +74,15 @@ oppia.directive('oppiaInteractiveEndExploration', [
         if (!$scope.isInEditorPreviewMode) {
           authorRecommendationsPromise.then(function() {
             var explorationId = explorationContextService.getExplorationId();
+            var collectionSuffix = '';
+            if ($scope.collectionId) {
+              collectionSuffix = '?collection_id=' + $scope.collectionId;
```

Comment:
I just found out from a commit @AllanYangZhou did that $http.get allows passing 'params':

    https://github.com/oppia/oppia/pull/1070/files#diff-f8104c948e4f0e8e89c3996704941aeeR38

Maybe we can use that?